aten/src/ATen/native/native_functions.yaml

```diff
@@ -5296,6 +5296,15 @@
 - func: _fake_quantize_learnable_per_channel_affine_backward(Tensor grad, Tensor self, Tensor scale, Tensor zero_point, int axis, int quant_min, int quant_max, float grad_factor=1.0) -> (Tensor, Tensor, Tensor)
   variants: function
 
+- func: fused_moving_avg_obs_fake_quant(Tensor(a) self, Tensor observer_on, Tensor fake_quant_on, Tensor averaging_const, Tensor(a!) running_min, Tensor(b!) running_max, Tensor(c!) scale, Tensor(d!) zero_point, int quant_min, int quant_max, int ch_axis, bool per_row_fake_quant=False, bool symmetric_quant=False) -> Tensor
```

Comment:
To indicate that `self` is aliased with the output you could write: `fn_name(Tensor(a) self) -> Tensor(a)`.
You might want to shift `Tensor(a!) running_min, Tensor(b!) running_max, Tensor(c!) scale, Tensor(d!)` these letters down to `b, c, d, e`.

See https://github.com/pytorch/pytorch/tree/master/aten/src/ATen/native